js/api/Triggers.js

```diff
@@ -60,6 +59,9 @@ export function checkTimeTriggers(omitNotifications) {
         checkSurveyTimeTriggers(surveys[i], omitNotifications);
       }
     }
+    if (callback) {
+      callback(null);
```

Comment:
Why callback `null`?  If this nodeJs style callback of (err, res) and there isn't an error or response I'd think it would be sufficient to do `callback()` as err would be undefined and a falsy value.